Index: lesson_4/hometask_04.7.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Task 7\r\n\r\nfrom math import factorial\r\n\r\nn = int(input(\"Введите число: \"))\r\ndef fact():\r\n    for i in range(0, n + 1):\r\n        a = factorial(i)\r\n        yield f'{i}! = {a}' # подсмотрел дял читаемости по красивее\r\n\r\nfor el in fact():\r\n    print(el)\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lesson_4/hometask_04.7.py b/lesson_4/hometask_04.7.py
--- a/lesson_4/hometask_04.7.py	(revision 3fc055fe55708f3239db0dbd37ff80168e0de769)
+++ b/lesson_4/hometask_04.7.py	(date 1641765743789)
@@ -1,13 +1,1 @@
 # Task 7
-
-from math import factorial
-
-n = int(input("Введите число: "))
-def fact():
-    for i in range(0, n + 1):
-        a = factorial(i)
-        yield f'{i}! = {a}' # подсмотрел дял читаемости по красивее
-
-for el in fact():
-    print(el)
-
