Index: lesson_4/hometask_04.6.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Task 6\r\n\r\nfrom itertools import count, cycle\r\n\r\n# COUNT\r\nfor el in count(1):\r\n    if el == 15:\r\n        break\r\n    else:\r\n        print(el)\r\n\r\n# CYCLE\r\nc = 0\r\nfor el in cycle(\"zero\"):\r\n    if c > 7:\r\n        break\r\n    print(el)\r\n    c += 1\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lesson_4/hometask_04.6.py b/lesson_4/hometask_04.6.py
--- a/lesson_4/hometask_04.6.py	(revision 3fc055fe55708f3239db0dbd37ff80168e0de769)
+++ b/lesson_4/hometask_04.6.py	(date 1641767528815)
@@ -1,18 +1,1 @@
-# Task 6
-
-from itertools import count, cycle
-
-# COUNT
-for el in count(1):
-    if el == 15:
-        break
-    else:
-        print(el)
-
-# CYCLE
-c = 0
-for el in cycle("zero"):
-    if c > 7:
-        break
-    print(el)
-    c += 1
+# Task 6
\ No newline at end of file
